src/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/08b4cd9ea0.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Home Page</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo-container">
+          <h3 id="logo">My Logo</h3>
+        </div>
+        <label for="check" class="hamberger-btn">
+          <i id="ham" class="fas fa-bars"></i>
+        </label>
+      </nav>
```

Comment:
- Kindly make sure that the container of the navigation barre section is in the `header` tag. :+1: 

* Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.The container of the section is a header tag.


- Kindly make sure the logo (if the design has one) has a link (`a` tag). :+1: 

   * This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
   
- and also the hamburger button has a `nav` tag.

   * The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices. The logo (if the design has one) is on the left and has a link (a tag).

```
eg.   
     
   <header class="container">
        <a href="#"> My Logo</a>
        <nav for="check" class="hamberger-btn">
          <i id="ham" class="fas fa-bars"></i>
        </nav>
    </header>

```